apps/SampleSite/PopupSamples/modal.html

```diff
@@ -5,12 +5,32 @@
         <!--The following src path should point to your local RequireJS config.js file-->
         <script type="text/javascript" src="config.js"></script>
         <script>
-            require(["apps/SampleSite/tempsrc/Popup", "src/amchart/Polar", "src/layout/Surface"], function (Popup, Polar, Surface) {
+            require(["src/layout/Popup", "src/amchart/Polar", "src/layout/Surface"], function (Popup, Polar, Surface) {
                 var modal = new Popup()
                     .size({width:500,height:500})
                     .target("main")
                     .position("fixed")
-                    .widget(new Surface().title("Modal Dialog Example").widget(new Polar().testData()).surfaceBackgroundColor("rgb(234, 249, 255)"))
+                    .widget(new Surface()
+                        .title("Modal Dialog Example")
+                        .surfaceBackgroundColor("rgb(234, 249, 255)")
+                        .widget(new Polar()
+                            .columns(["Subject", "Year 1", "Year 2", "Year 3", "Year 4"])
```

Comment:
Per our conversation yesterday, its better for the users if they can see the data/column functions and be able to play with the data.